src/core_modules/capture-core/components/Pages/EnrollmentEvent/EnrollmentEventPage.container.js

```diff
@@ -18,12 +23,33 @@ export const EnrollmentEventPage = () => {
         }),
         shallowEqual,
     );
-    const [mode] = useState(pageMode.VIEW);
     const { program } = useProgramInfo(programId);
     const programStage = [...program.stages?.values()].find(
         item => item.id === stageId,
     );
+
     return (
-        <EnrollmentEventPageComponent mode={mode} programStage={programStage} />
+        <EnrollmentEventPageComponent
+            mode={showEditEvent ? pageMode.EDIT : pageMode.VIEW}
+            programStage={programStage}
+            onEdit={() => onOpenEditEvent()}
+        />
     );
 };
+
+const mapStateToProps = (state: ReduxState) => ({
+    showEditEvent: state.viewEventPage?.eventDetailsSection?.showEditEvent,
```

Comment:
Maybe we can move the location of the state here to something like `state.widgetEventEdit`?

Reply:
If I change anything in the Redux state path then the old redux actions and reducers will not work anymore. Therefore if we don't want a big refactor I think is best to use the same path that are being used in the single event form

Reply:
Ok, let's leave it for now.